### PR TITLE
[CEN-1439] Add load test RTD storage

### PR DESCRIPTION
**Pull request by alberto-re-pagopa**
This PR introduces a load test for the PUT endpoint of the /pagopastorage resource. The rationale behind this test is to verify that:
a) from the client(s) perspective multiple uploads in a short timespan just work fine (201)
b) the decrypt service is capable to handle all the submitted files

Obviously this test just takes care of verifying 'a', that 'b' is also working correctly needs manual logs surveillance.

### List of changes

- new load test for RTD Storage API

### Motivation and context

- verify that the upload chain works correctly even when many files are uploaded concurrently and/or in a short timespan

### Aggregation level

- [x] Test case
- [ ] Test suite

### Affected environment

- [x] Development (DEV)
- [x] User Acceptance / Third Part Integration (UAT)
- [ ] Production (PROD)

### Test type

- [ ] End to end
- [x] Performance
- [ ] Smoke test

### Type of changes

- [x] Add new test case/suite
- [ ] Modify existing test case/suite

### Test Results

- [x] Success
- [ ] Failure
- [ ] Poor performance
- [ ] Good performance

### CODE REVIEW

File: test/performance/rtdStorage.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+		{
+			domains: [baseUrl],
+			cert: open(`../../certs/${myEnv.MAUTH_CERT_NAME}`),
+			key: open(`../../certs/${myEnv.MAUTH_PRIVATE_KEY_NAME}`),
+		},
+	]
+
+	params.headers = {
+		'Ocp-Apim-Subscription-Key': myEnv.APIM_RTDPRODUCT_SK,
+		'Ocp-Apim-Trace': 'true',
```
Comment (GiovanniMancini) [inline test/performance/rtdStorage.js:37]: Maybe a performance test doesn't need to be traced. It could be just impossible due to the number of requests

Comment (alberto-re-pagopa) [reply test/performance/rtdStorage.js:37]: Nice catch, I'll fix it

File: test/performance/rtdStorage.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+					blob,
+					payload,
+					myEnv.RTD_STORAGE_SAS
+				),
+				[statusCreated()]
+			)
+		)
+	})
+
+	n += 1
```
Comment (GiovanniMancini) [inline test/performance/rtdStorage.js:79]: This is not so clear to me. I barely understand that it allows to reiterate the test :-), but it was supposed to be done via configuration

**Review by GiovanniMancini — APPROVED** [review]

File: test/performance/rtdStorage.js (hunk 2 above)
Comment (alberto-re-pagopa) [reply test/performance/rtdStorage.js:79]: You're right, I'll better add a comment to the test. The rationale here is to be able to generate files each with a different name, thus allowing to compare the number of 201 uploaded generated by this test with the presence in the rtd-ms-decrypted container of exactly N files. This poses some limit to the effective scaling of this test (no more than 9 VUs, no more than 99 seconds of execution to guarantee that each file(name) will be uploaded once) but given the expected TPS of this endpoint I think is more than enough.

**Review by GiovanniMancini — APPROVED** [review]

**Pull request merged**